py-polars/tests/unit/test_cfg.py

```diff
@@ -415,6 +415,7 @@ def test_string_cache() -> None:
     assert_frame_equal(out, expected)
 
 
+@pytest.mark.xdist_group(name="group1")
```

Comment:
@stinodego I marked `"group1"` as the group for any contention. E.g. things depending on global env vars should go to this group.

Reply:
I think you can just use monkeypatch for the environment variable, and everything should run fine in parallel? Every test should be able to run in isolation.

Reply:
Moneypatch will not set that variable in rust. Env vars are global.

Reply:
Just merge it like this, I'll take a look at this later to see if there's a better way!